packages/@aws-cdk/aws-rds/lib/cluster-parameter-group.ts

```diff
@@ -0,0 +1,70 @@
+import cdk = require('@aws-cdk/cdk');
+import { ClusterParameterGroupRef } from './cluster-parameter-group-ref';
+import { Parameters } from './props';
+import { cloudformation, DBClusterParameterGroupName } from './rds.generated';
+
+/**
+ * Properties for a cluster parameter group
+ */
+export interface ClusterParameterGroupProps {
+    /**
+     * Database family of this parameter group
+     */
+    family: string;
+
+    /**
+     * Description for this parameter group
+     */
+    description: string;
+
+    /**
+     * The parameters in this parameter group
+     */
+    parameters?: Parameters;
+}
+
+/**
+ * Defina a cluster parameter group
+ */
+export class ClusterParameterGroup extends ClusterParameterGroupRef {
+    public readonly parameterGroupName: DBClusterParameterGroupName;
+    private readonly parameters: Parameters = {};
+
+    constructor(parent: cdk.Construct, id: string, props: ClusterParameterGroupProps) {
+        super(parent, id);
+
+        const resource = new cloudformation.DBClusterParameterGroupResource(this, 'Resource', {
+            description: props.description,
+            family: props.family,
+            parameters: new cdk.Token(() => this.parameters),
+        });
+
+        for (const [key, value] of Object.entries(props.parameters || {})) {
+            this.setParameter(key, value);
+        }
+
+        this.parameterGroupName = resource.ref;
+    }
+
+    /**
+     * Set a single parameter in this parameter group
+     */
+    public setParameter(key: string, value: string | undefined) {
```

Comment:
Should we also add `removeParameter`?

Reply:
I thought `setParameter(name, undefined)` would fulfil that purpose.

Reply:
Let's just sprinkle some sugar on top of it. Wouldn't be my first guess when looking to remove a parameter 😉